test/resque_test.rb

```diff
@@ -12,7 +12,8 @@ class ResqueTest < MiniTest::Spec
   it "enqueues normal jobs" do
     Resque.enqueue FakeJob, "x"
     Resque.enqueue FakeJob, "x"
-    assert_equal 2, Resque.size(:normal)
+    queue_name = FakeJob.instance_variable_get(:@queue)
```

Comment:
Sure, should be good now I think...  I haven't yet figured out the workflow to rebase and squash commits yet so let me know if I need to figure out how to make that happen.